website/example.css

```diff
@@ -0,0 +1,163 @@
+body, html {
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    font-family: Avenir Next;
```

Comment:
You are never importing this font as it is a paid font, and setting no fallback defaults to standard on browsers and computers without it. Consider using [Nunito Sans](https://fonts.google.com/specimen/Nunito+Sans) from google fonts as it's is visually very similar and can be expected to display the same everywhere.